core/src/main/java/com/zextras/carbonio/files/dal/repositories/interfaces/UserRepository.java

```diff
@@ -5,22 +5,19 @@
 package com.zextras.carbonio.files.dal.repositories.interfaces;
 
 import com.zextras.carbonio.files.dal.dao.User;
+import com.zextras.carbonio.files.dal.dao.UserMyself;
 import com.zextras.carbonio.usermanagement.entities.UserId;
 import io.vavr.control.Try;
 import java.util.Optional;
 
 public interface UserRepository {
 
-  Optional<User> getUserById(
-    String cookies,
-    String userId
-  );
+  // no cache on this one since it is not requested often and we always want the updated version
```

Comment:
Update the comment to a javadoc describing the input parameters and the return object